packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -200,10 +201,8 @@ function doMoveAll(
         case "polygon": {
             let newCoords: vec.Vector2[];
             switch (state.snapTo) {
-                // TODO(LEMS-1902): Implement sides snapping for polygons
                 case "grid":
-                case undefined:
-                case "sides": {
+                case undefined: {
```

Comment:
Would `case undefined` be better handled by a default for the switch case?

Reply:
Hmm, I'm not sure what the best way to handle it would be. It should use the exact same bit of code as grid, so that's why I put it there, but maybe it would be better as a default since something other than undefined might be passed and we want it to always use the grid options for that. Thanks! I'll change it :)

Reply:
Should I just not have a case for grid then and have anything other than angles and sides go to default? Or would that be confusing?